docs/deploying/deploy-faucets.mdx

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 3
+toc_max_heading_level: 5
+---
+
+# Faucets (Available)
+
+Faucets for deployment of smart contracts can be requested from the website.
+
+The button is at the bottom left of the  [Demo app] (https://scaffold-stark-demo.vercel.app/debug)
+
+
+![Faucets Example](/img/faucets.jpg)
+
+On clicking on it, You will get a pop up
+
+![Sepolia Example](/img/Sepolia.jpg)
+
+
+## Websites available for Starknet Sepolia faucets
```

Comment:
We also need some docs on the Devnet faucet, which looks different from this sepolia window. The devnet one allows user to fill in their own wallet with some tokens

Reply:
For this section, we can just say "Starknet Sepolia Faucets"

Reply:
@pheobeayo I meant the title could just be "Starknet Sepolia Faucets" instead of "Websites available ...."

Reply:
and can remove the sub "Starknet Sepolia Faucets"

Reply:
Also for the devnet, please indicate the UI path also. as shown by @Nadai2010